lua/lspconfig/server_configurations/htmx.lua

```diff
@@ -0,0 +1,27 @@
+local util = require 'lspconfig.util'
+-- NOTE: the binary can be installed in custom a location. The user should
+-- change the cmd to match the installation path
+local cargo_home = os.getenv 'CARGO_INSTALL_ROOT' or os.getenv 'CARGO_HOME' or os.getenv 'HOME' .. '/.cargo'
+
+return {
+  default_config = {
+    cmd = { cargo_home .. '/bin/htmx-lsp' },
```

Comment:
This doesn't make sense. If I install the binary somewhere else. will fail here

Reply:
That is correct. Pushed a change to call without the path.